.github/workflows/docker-hub-release.yml

```diff
@@ -0,0 +1,45 @@
+# Based on https://github.com/matrix-org/dendrite/blob/master/.github/workflows/docker-hub.yml
+
+name: "Docker Hub - Release"
+
+on:
+  release:
+    types: [published]
+
+env:
+  DOCKER_NAMESPACE: matrixdotorg
+  PLATFORMS: linux/amd64
+
+jobs:
+  docker-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Get release tag
+        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+      - name: Login to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_HUB_USERNAME }}
+          password: ${{ secrets.DOCKER_HUB_TOKEN }}
+
+      - name: Build image 'release'
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: ./Dockerfile
+          platforms: ${{ env.PLATFORMS }}
+          push: true
+          tags: |
+            ${{ env.DOCKER_NAMESPACE }}/matrix-appservice-slack:release-${{ env.RELEASE_VERSION }}
+
+      - name: Build image 'stable'
```

Comment:
Can this not be combined with `Build image 'release'` by handing it multiple tags?

Reply:
It now has :)